applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1542,43 +1543,35 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateExtrapolationMatrix(Bounde
 template <>
 void UPwSmallStrainElement<2, 3>::CalculateExtrapolationMatrix(BoundedMatrix<double, 3, 3>& rExtrapolationMatrix)
 {
-    // The matrix contains the shape functions at each GP evaluated at each
-    // node. Rows: nodes Columns: GP
-
     // Triangle_2d_3
     // GI_GAUSS_2
```

Comment:
If the template will be used, then these comments are obsolete. If not, I would suggest to use constexpr for dimension instead //Triangle_2d_3

Reply:
Removed.